library/src/main/java/com/novoda/downloadmanager/BatchBuilder.java

```diff
@@ -20,4 +20,12 @@
      * @return an instance of {@link Batch}.
      */
     Batch build();
+
+    static BatchBuilder from(Batch batch) {
```

Comment:
Maybe do you want to add a javadoc description?

Reply:
Ah true 😂